indexers/vector/AnnoyIndexer/README.md

```diff
@@ -1,3 +1,66 @@
 # AnnoyIndexer
 
-Annoy powered vector indexer 
\ No newline at end of file
+Annoy (Approximate Nearest Neighbors Oh Yeah) is a C++ library with Python bindings to search for points in space that are close to a given query point. It also creates large read-only file-based data structures that are mmapped into memory so that many processes may share the same data.
+AnnoyIndexer hence is an Annoy powered vector indexer.
+For more information, refer to the GitHub repo for [Spotify's Annoy](https://github.com/spotify/annoy).
+
+## Snippets:
+
+Initialise AnnoyIndexer:
+
+`AnnoyIndexer(hostname=host_ip, port=host_port, db=db_int)`
```

Comment:
AnnoyIndexer does not have these parameters.

Reply:
Resolved to the right params